src/snowflake/connector/pandas_tools.py

```diff
@@ -49,6 +50,7 @@ def write_pandas(
     auto_create_table: bool = False,
     create_temp_table: bool = False,
     overwrite: bool = False,
+    table_type: str = "",
```

Comment:
Could we introduce a unique [enum](https://docs.python.org/3/library/enum.html#creating-an-enum). Otherwise this should have more precise type-hints like `typing.Literal["", "temp", "temporary", "transient"]`

Reply:
I considered using `enum` in the snowpark but @sfc-gh-yixie suggested that using a string is more user friendly when discussing about similar changes in snowpark. I'll add a more precise type-hints (and make same change in snowpark as well). Thanks for catching this.

Reply:
Hmmmm `Literal` is only available from 3.8 and above (https://docs.python.org/3/library/typing.html#typing.Literal). So I'm not sure if we could do this given we still support 3.7 :(

Reply:
Discussed offline with @sfc-gh-mkeller , this is indeed caused by unable to import `Literal`. We import from `typing_extensions` instead.